Aztec/Classes/Extensions/NSAttributedString+Encoding.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+// MARK: - NSAttributedString Enconding methods
+//
+extension NSAttributedString
+{
+    static let pastesboardUTI = "com.wordpress.aztec.attributedString"
+
+    func archiveToData() -> Data {
+        let data = NSKeyedArchiver.archivedData(withRootObject: self)
+        return data
+    }
+
+    static func unarchive(fromData data: Data) -> NSAttributedString? {
```

Comment:
In Swift 3 style this would probably look like...

```
static func unarchive(from data: Data) -> NSAttributedString?
```

(No need to change anything though!!)